wolfssl/wolfcrypt/asn_public.h

```diff
@@ -772,6 +772,10 @@ WOLFSSL_API int  wc_ParseCert(struct DecodedCert*, int, int, void*);
 WOLFSSL_API int wc_GetPubKeyDerFromCert(struct DecodedCert* cert,
                                         byte* derKey, word32* derKeySz);
 
+typedef time_t (*wolfSSL_time_cb)(time_t* t);
```

Comment:
Can you put these near `wc_GetTime`? Should this all be wrapped with `NO_ASN_TIME`?

Reply:
Sure, seems reasonable.